rudder-lang/tests/helpers/cfjson_tester

```diff
@@ -4,20 +4,29 @@
 Ncf / rudder-lang compatibility tester
 
 Usage: 
-    ncf ncf-to-json <file.cf> <new.json>
-    ncf compare-json <first.json> <new.json>
-    ncf compare-cf <first.cf> <new.cf>
+    ncf ncf-to-json <file.cf> <new.json> [<local_config_path.conf>]
```

Comment:
optional parameters should be prefixed with an option name like in "-c file.conf"

Reply:
I wanted to but the parameters parser does not handles "-options" correctly